packages/gatsby-plugin-mdx/README.md

```diff
@@ -417,6 +418,27 @@ Gatsby includes the media-type of the content on any given node. For
 by the file extension. For remote content or generated content, we
 choose which nodes to process by looking at the media type.
 
+#### Time to read
+
+Calculating the time to read a Markdown document based on the word
+count or Markdown node. This is useful for customizing the time to
+read a document based on a faster or slower words per minute reading
+rate or by custom heuristics based on the markdown node.
+
+```js
+// gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-mdx`,
+      options: {
+        timeToRead: (wordCount, mdxNode) => wordCount / 42,
```

Comment:
not match `timeToRead(wordCount, html, md)`

Reply:
Updated 👍